.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ on:
   pull_request:
     branches: [main]
     paths:
+      - ".ci/**"
```

Comment:
Hah, good call.

Reply:
I'm thinking of removing `branches: [main]` as well, that way PR's to PR's will also run CI which would be great

Reply:
Sounds good! In a separate commit?

Reply:
Yep - just squashing them

Reply:
I meant, would be good to have the `test.yml` changes in a separate commit from the `Cargo.toml` changes since they're conceptually independent.

Reply:
Good point, will push up two separate commits